tests_tf/test_attacks.py

```diff
@@ -974,6 +891,40 @@ def test_generate_np_gives_clipped_adversarial_examples(self):
     self.assertLess(-0.201, np.min(x_adv))
     self.assertLess(np.max(x_adv), .301)
 
+
+  def test_generate_np_does_not_cache_graph_computation_for_nb_iter(self):
+    x_val = np.random.rand(100, 2)
+    x_val = np.array(x_val, dtype=np.float32)
+
+    x_adv = self.attack.generate_np(x_val, eps=1.0, ord=np.inf,
+                                    clip_min=-5.0, clip_max=5.0,
+                                    nb_iter=10)
+
+    orig_labs = np.argmax(self.sess.run(self.model.get_logits(x_val)), axis=1)
+    new_labs = np.argmax(self.sess.run(self.model.get_logits(x_adv)), axis=1)
+    self.assertTrue(np.mean(orig_labs == new_labs) < 0.1)
```

Comment:
should be assertLess

Reply:
[these were from the copied code and were already tests]